react-components/src/library/components/stem-finder-result.js

```diff
@@ -94,10 +94,18 @@ const StemFinderResult = Component({
     }.bind(this)
     if (resource.is_favorite) {
       jQuery.post('/api/v1/materials/remove_favorite', { favorite_id: resource.favorite_id }, done)
-      ga('send', 'event', 'Favorite Button', 'Click', `${resource.name} removed from favorites`)
+      gtag('event', 'click', {
+        'category': 'Favorite Button',
+        'action': 'opened',
+        'resource': `${resource.name} removed from favorites`,
+      });
     } else {
       jQuery.post('/api/v1/materials/add_favorite', { id: resource.id, material_type: resource.class_name_underscored }, done)
-      ga('send', 'event', 'Favorite Button', 'Click', `${resource.name} added to favorites`)
+      gtag('event', 'click', {
+        'category': 'Favorite Button',
+        'action': 'opened',
```

Comment:
This `'action': 'opened',` doesn't seem to match up anything in the previous event tracking code. Did you mean to add it?

If not, it should be removed, as should the other instance on line 99.

Reply:
I don't understand why these `'action':` params are still here. The action is `click` which is already set as the second value after `event`. Please remove all instances of this or explain why you added the additional `action` parameter. Thanks.